src/Dom/Document.php

```diff
@@ -880,7 +880,7 @@ private function replaceSelfClosingTags($html)
         static $regexPattern = null;
 
         if (null === $regexPattern) {
-            $regexPattern = '#<(' . implode('|', Tag::SELF_CLOSING_TAGS) . ')([^>]*?)(?>\s*\/)?>(?!</\1>)#';
+            $regexPattern = '#<(' . implode('|', Tag::SELF_CLOSING_TAGS) . ')([^>]*?)(?>\s*(\/|\\\/))?>(?!</\1>)#';
```

Comment:
Here's an alternative proposal:

What if the pattern is updated to prevent matching self-closing slashes that are preceded by a backslash?

```suggestion
            $regexPattern = '#<(' . implode('|', Tag::SELF_CLOSING_TAGS) . ')([^>]*?)(?>\s*(?<!\\\\)\/)?>(?!</\1>)#';
```

Reply:
LGTM, that still fix the issue and without altering the jsonld markup 🎉